shared/profile/user/index.tsx

```diff
@@ -352,7 +354,7 @@ class User extends React.Component<Props, State> {
       } else {
         chunks.push({
           text: 'Loading...',
-          type: 'loading',
+          type: 'loading' as 'loading',
```

Comment:
It would be nice to update our eslint so we can use `'loading' as const` here instead